### ISSUE TITLE
Crash when going up the stairs

### ISSUE THREAD

**Issue by nowakowsk** [Elatronion/televoid#2]
I was going up the stairs (for way too long) on the screen with the flower, sign, glue and elevator. The game crashed eventually.

**Version:** 10.8.1
**Platform:** Windows 10 64-bit

Console output:
```
Adding Scene Entity "floor"
[HGE LOG] src/Scenes/Scene.c:459: Parsing TMX Layer "props".
[HGE LOG] src/Scenes/Scene.c:463:       OBJECT LAYER
[HGE LOG] src/Scenes/Scene.c:411: Parsing TMX Object "glue" type "prop"...
[HGE WARNING] src/Scenes/Scene.c:307: File "res/textures/sprites/props/glue_normal.png" doesn't exists. Using default.
Adding Scene Entity "glue"
[HGE LOG] src/Scenes/Scene.c:411: Parsing TMX Object "flower" type "prop"...
[HGE WARNING] src/Scenes/Scene.c:307: File "res/textures/sprites/props/flower_normal.png" doesn't exists. Using default.
Adding Scene Entity "flower"
[HGE LOG] src/Scenes/Scene.c:459: Parsing TMX Layer "trigger".
[HGE LOG] src/Scenes/Scene.c:463:       OBJECT LAYER
[HGE LOG] src/Scenes/Scene.c:411: Parsing TMX Object "Going Down" type "trigger"...
Adding Scene Entity "trigger"
[HGE LOG] src/Scenes/Scene.c:411: Parsing TMX Object "Going Up" type "trigger"...
Adding Scene Entity "trigger"
[HGE LOG] src/Scenes/Scene.c:411: Parsing TMX Object "SPLURP" type "trigger"...
Adding Scene Entity "trigger"
[HGE LOG] src/Scenes/Scene.c:411: Parsing TMX Object "SPLURP" type "trigger"...
Adding Scene Entity "trigger"
[HGE LOG] src/Scenes/Scene.c:459: Parsing TMX Layer "spawn".
[HGE LOG] src/Scenes/Scene.c:463:       OBJECT LAYER
[HGE LOG] src/Scenes/Scene.c:411: Parsing TMX Object "DEFAULT" type "player_start"...
player_start:
        previous_scene_that_activates: ''
        fadein_on_start: 1
        fade_speed: 1.234000
Adding Scene Entity "ian"
[HGE LOG] src/Scenes/Scene.c:607: Last Loaded Scene: "res/scenes/Stairs/Stairs UP.tmx"
trigger->exec_flags : 2
```

**Comment by Elatronion** [comment]
I'm looking into it! :)
It might be related to another crash when loading door 100. (someone else had)

**Comment by Elatronion** [comment]
With the help of @ThomasWorkOnline most if not all crashes should be resolved in the latest build (11.1.0).
This build is available on [GameJolt](https://gamejolt.com/games/Televoid_out_of_the_loop/377460) and, after some tests, the code will be pushed.
